src/main/java/org/terasology/launcher/LauncherInitTask.java

```diff
@@ -132,18 +131,15 @@ protected LauncherConfiguration call() {
         return null;
     }
 
-    private OperatingSystem getOperatingSystem() throws LauncherStartFailedException {
-        logger.trace("Init OperatingSystem...");
+    private Platform getPlatform() throws LauncherStartFailedException {
```

Comment:
Since you removed `throw new LauncherStartFailedException();` this method should not throw such an exception anymore.

```suggestion
    private Platform getPlatform() {
```

Reply:
godd catch, missed that 😅